cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -386,6 +385,7 @@ impl OpenshiftSecondaryMetadataParserPlugin {
                         let mut channels_split = channels.split(',').collect::<Vec<_>>();
                         // this has to match the sorting at
                         // https://github.com/openshift/cincinnati-graph-data/blob/5fc8dd0825b42369de8070ecba2ae0c49d0a99d9/hack/graph-util.py#L187
+                        channels_split.sort_by(|a, b| a.cmp(b));
```

Comment:
I think it's possible to merge the two `sort_by` calls here using [Ordering::then](https://doc.rust-lang.org/std/cmp/enum.Ordering.html#method.then).

I imagine something like this (untested code):

```rust
channels_split.sort_by(|a, b| {
    let a_split: Vec<&str> = a.splitn(2, '-').collect();
    let b_split: Vec<&str> = b.splitn(2, '-').collect();

    # sort first by channel: prerelease, stable, fast, candidate, etc
    let cmp_channel = a.cmp(b); 

    # sort by version: 4.1, 4.2, etc
    let cmp_version = a_split[1].cmp(b_split[1]);

    cmp_channel.then(cmp_version)
}
```